### ISSUE TITLE
TPAC 2022 status report

### ISSUE THREAD

**Issue by LJWatson** [w3c/gamepad#169]
[TPAC 2022 ](https://www.w3.org/2022/09/TPAC/) is hybrid this year. WebApps will meet in Vancouver on [12 and 13 September](https://www.w3.org/calendar/tpac2022/). Please [register](https://www.w3.org/2022/09/TPAC/registration.html) if you plan to attend in person or remotely.

Editors, we need you to do two things before 30 August 2022:

1. If you want agenda time to discuss your spec(s), please use W3C/webappswg#87 to request it
2. Complete a status report for your spec(s)

#### Status report

Your status report should answer these questions (as a comment on this issue):

1. What progress has your spec made in the last 12 months?
2. Is anything blocking your spec from moving to CR/Rec?
3. If yes, what is your plan to unblock it and do you need any help?

Please post a status report on your spec(s) even if you do not plan to attend TPAC at all.

CC @marcoscaceres and @siusin

**Comment by nondebug** [comment]
> 1. What progress has your spec made in the last 12 months?

[Add support for Xbox One impulse trigger effects.](https://github.com/w3c/gamepad/issues/138) We added a "trigger-rumble" effect that includes vibration intensity for the left and right trigger actuators found on Xbox One gamepads in addition to the normal left and right rumble motors. We also revised the way vibration capability detection works by adding a canPlay() method that returns true if the effect type is supported by the actuator.

[Support trackpad inputs.](https://github.com/w3c/gamepad/issues/34) We are working on adding support for multitouch surfaces like those found on PlayStation gamepads.

> 2. Is anything blocking your spec from moving to CR/Rec?

We will need to decide how to handle back-forward cache integration:
[gamepadconnected/disconnected events & non-"fully active" documents](https://github.com/w3c/gamepad/issues/149)

> 3. If yes, what is your plan to unblock it and do you need any help?

We plan to discuss this at our breakout session.
